### ISSUE TITLE
android_device_samsung_star2lte-android-9.0.zip

### ISSUE THREAD

**Issue by brmccarta1** [koalaman/shellcheck#2072]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC1073, SC1065, SC1064, SC1072
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
javascript:alert('Are you on mobile and unable to paste in the editor? Paste in this plaintext box to have it copied over.')
```

#### Here's what shellcheck currently says:
```
Line 1	SC1073: Couldn't parse this function. Fix to allow more checks.
Line 1	SC1065: Trying to declare parameters? Don't. Use () and refer to params as $1, $2..
Line 1	SC1064: Expected a { to open the function definition.
Line 1	SC1072:  Fix any mentioned problems and try again.
```
#### Here's what I wanted or expected to see:

TODO: Describe expected/desired output
